db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
```

Comment:
Per UPS brand guidelines, can we place the logo on white background? 

![CleanShot 2024-02-02 at 12 30 11@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/a9b0362d-cf6b-4759-a538-78fccaa54c2a)